enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/AccessLoggingService.java

```diff
@@ -76,7 +79,8 @@ public void onNext(StreamAccessLogsMessage message) {
 
             @Override
             public void onError(Throwable throwable) {
-                logger.error("Error while receiving access log entries from router. " + throwable.getMessage());
+                logger.error("Error while receiving access log entries from router. " + throwable.getMessage(),
+                        errorLog(LoggingConstants.Severity.CRITICAL, 5101));
```

Comment:
Is this error codes documented somewhere

Reply:
Yes, we are maintaining a list of error codes along with some description, separately for enforcer and adapter..